apps/src/code-studio/components/StatusCheckmarkIcon.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import color from '@cdo/apps/util/color';
+import {makeEnum} from '@cdo/apps/utils';
+import PropTypes from 'prop-types';
+
+export const iconStatus = makeEnum('noAction', 'success', 'failure');
+
+export default function StatusCheckmarkIcon(props) {
```

Comment:
this component isn't always a checkmark (sometimes it's an X) -- should this be something like `StatusIcon` instead?